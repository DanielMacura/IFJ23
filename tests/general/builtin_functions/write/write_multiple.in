write("Hello ", 2, "nd World, or is it ", 3.33321,"rd?")
