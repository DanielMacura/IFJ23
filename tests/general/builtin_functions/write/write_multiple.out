Hello 2nd World, or is it 0x1.aaa6a012599edp+1rd?